src/index.css

```diff
@@ -72,5 +72,6 @@
     }
     body {
       @apply bg-background text-foreground;
+      overflow-y: hidden;
     }
   }
```

Comment:
Would be nice to have empty line in the end of files. Github shows this red icon on purpose 😉 

Worth to read:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Added empty line.